modules/luceadBidAdapter.js

```diff
@@ -6,16 +6,18 @@ import {fetch} from '../src/ajax.js';
 
 const bidderCode = 'lucead';
 const bidderName = 'Lucead';
-let baseUrl = 'https://lucead.com';
-let staticUrl = 'https://s.lucead.com';
-let companionUrl = 'https://cdn.jsdelivr.net/gh/lucead/prebid-js-external-js-lucead@master/dist/prod.min.js';
-let endpointUrl = 'https://prebid.lucead.com/go';
 const defaultCurrency = 'EUR';
 const defaultTtl = 500;
 const aliases = ['adliveplus'];
+const defaultRegion = 'eu';
+const domain = 'lucead.com'
+let baseUrl = `https://${domain}`;
+let staticUrl = `https://s.${domain}`;
+let endpointUrl = baseUrl;
+let companionUrl = 'https://cdn.jsdelivr.net/gh/lucead/prebid-js-external-js-lucead@master/dist/prod.min.js';
```

Comment:
this is not allowed, you have to delete this. See #10653

Reply:
removed.